app/api/chemotion/research_plan_api.rb

```diff
@@ -68,19 +68,22 @@ class ResearchPlanAPI < Grape::API
         optional :collection_id, type: Integer, desc: 'Collection ID'
```

Comment:
I would not reuse the API endpoint for creating but instead create a new one which only handles the copying of a RP to avoid increasing of complexity

Reply:
This is to maintain consistency of the similar use copying methods of samples & reactions. Refactor and separate API points can be created for all of them as another task.